### PR TITLE
added code syntax language and a few readability improvements.

### PR DESCRIPTION
**Pull request by johnmwright**
## Summary

Continuing the cleanup of WCF pages (see #4991 and #8128), this time focusing on the [WFC:Admin & Diagnostics](https://docs.microsoft.com/en-us/dotnet/framework/wcf/diagnostics/) and [WFC:Extending WFC](https://docs.microsoft.com/en-us/dotnet/framework/wcf/extending/) areas.

## Changes:
- Applying `csharp` syntax highlighting where it was missing.
- Cleaned up a few spacing issues.

### CODE REVIEW

File: docs/framework/wcf/diagnostics/wmi/messageencodingbindingelement.md [hunk 1]
```diff
@@ -7,13 +7,12 @@ ms.assetid: 7f750742-b96b-498f-bf5e-05933a1a5961
...
-  
- `string MessageVersion;`  
-  
- `};`  
+```csharp
+class MessageEncodingBindingElement : BindingElement
+{
+    string MessageVersion;  
+};  
+ ```
```
Comment (rpetrusha) [inline docs/framework/wcf/diagnostics/wmi/messageencodingbindingelement.md]: This is a really nice catch!

File: docs/framework/wcf/diagnostics/wmi/peercustomresolverbindingelement.md [hunk 2]
```diff
@@ -7,15 +7,13 @@ ms.assetid: 9ccc2770-a20e-4dff-9970-f56ad8aec2b5
...
- `class PeerCustomResolverBindingElement : PeerResolverBindingElement`  
-  
- `{`  
-  
- `string Address;`  
-  
- `string Binding;`  
-  
- `};`  
+```csharp
```
Comment (rpetrusha) [inline docs/framework/wcf/diagnostics/wmi/peercustomresolverbindingelement.md]: again, nice catch!

File: docs/framework/wcf/extending/custom-encoders.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ This topic discusses how to create custom encoders.
...
+-   <<!--zz xref:System.ServiceModel.Channels.MTOMMessageEncodingBindingElement -->`System.ServiceModel.Channels.MTOMMessageEncodingBindingElement`>: Represents the binding element that specifies the character encoding and message versioning used for a message using a Message Transmission Optimization Mechanism (MTOM) encoding. MTOM is an efficient technology for transmitting binary data in WCF messages. The MTOM encoder attempts to balance between efficiency and interoperability. The MTOM encoding transmits most XML in textual form, but optimizes large blocks of binary data by transmitting them as-is, without conversion to text.
```
Comment (rpetrusha) [inline docs/framework/wcf/extending/custom-encoders.md]: @johnmwright, this is unrelated to your PR, but would you mind removing the comment and replacing the fenced API with the following link: \<xref:System.ServiceModel.Channels.MtomMessageEncodingBindingElement>?

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks great, @johnmwright. Besides adding language tags, you've also replaced some really ugly code formatting. I've left one request for a change that's not related to your PR. If you'd prefer, we can make the change.

File: docs/framework/wcf/extending/custom-encoders.md (hunk 3 above)
Comment (johnmwright) [reply docs/framework/wcf/extending/custom-encoders.md]: @rpetrusha Done!

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**